client/bio-commandline/src/test/kotlin/uk/ac/ebi/biostd/client/cli/BioStudiesCommandLineTest.kt

```diff
@@ -103,10 +103,11 @@ class BioStudiesCommandLineTest(
 
         every {
             mockWebClient.submitSingle(submissionFile, listOf())
-        } throws WebClientException(BAD_REQUEST, "Invalid Files")
+        } throws WebClientException(BAD_REQUEST, "<ValidationErrors><error>Invalid Files</error></ValidationErrors>")
 
         val exception = assertThrows<PrintMessage> { testInstance.parse(args) }
-        assertThat(exception.message).isEqualTo("Invalid Files")
+        assertThat(exception.message).isEqualTo(
```

Comment:
why xml error?

Reply:
That's the string I'm using in the mock response. It's intended to emulate what's returned by the server